charger/smartevse.go

```diff
@@ -293,3 +293,42 @@ func (wb *smartEVSE) Diagnose() {
 		fmt.Printf("\tCP interruption time: %dms\n", cptime)
 	}
 }
+
+
+var _ api.Resurrector = (*smartEVSE)(nil)
+
+// WakeUp implements the Resurrector interface
+func (wb *smartEVSE) WakeUp() error {
+	if wb.cphwonlock {
```

Comment:
@swallage these properties don't exist in the PR.

Reply:
Here I lack experience with Go. I received from another contributor the full patch, but you did the clean up in the intermediate time. Isn't the WakeUp a general feature of EVCC? In the previous template you removed the cphwonlock and cpwakeauto. Is that the property you are referrring to?

Reply:
In case the car supports EV40 the user can set that property via Modbus address 0204
0x10: automatic cp disconnect (ev40 or relais, handled by charger module
0x20: use ev40 for cp disconnect

Reply:
`wb.cphwonlock` does not exist. This PR does not compile.

Reply:
we used to have in the patch
+// smartEVSE is an api.Charger implementation
+type smartEVSE struct {
+       log        *util.Logger
+       conn       *modbus.Connection
+       curr       uint16
+       en         bool
+       cphwonlock bool
+       cpwakeauto bool
+       oldfw      bool
+}

Reply:
Both are missing here and need be added.